src/libraries/System.Security.Cryptography.Csp/src/System/Security/Cryptography/PasswordDeriveBytes.Windows.cs

```diff
@@ -27,7 +27,7 @@ public byte[] CryptDeriveKey(string algname, string alghashname, int keySize, by
             if (rgbIV == null)
                 throw new CryptographicException(SR.Cryptography_PasswordDerivedBytes_InvalidIV);
 
-            byte[] key = null;
+            byte[] key = null!;
```

Comment:
Why is this not nullable?

Reply:
The same, always set with non null in the method below `CapiHelper.DeriveKey(ProvHandle, algid, algidhash, _password, _password.Length, keySize << 16, rgbIV, rgbIV.Length, ref key)`, return value not expected to be null